src/Libraries/RevitNodesUI/Selection.cs

```diff
@@ -35,14 +35,48 @@ namespace Dynamo.Nodes
     public abstract class RevitSelection<TSelection, TResult> : SelectionBase<TSelection, TResult>
     {
         protected Document SelectionOwner { get; private set; }
-        protected RevitDynamoModel RevitDynamoModel { get; private set; }
+        private RevitDynamoModel revitDynamoModel;
 
         #region public properties
 
-        /* TODO: Now that nodes know nothing about their owners, we can't access RunEnabled.
+        public RevitDynamoModel RevitDynamoModel
+        {
+            get
+            {
+                return revitDynamoModel;
+            }
+            set
+            {
+                if (null != revitDynamoModel)
```

Comment:
I stopped doing this (as I have been advised few times before) quite some time back as it is no longer solving any real problem in C#, and instead I now do `revitDynamoModel != null`. So I decided to also advise you to not do this in the future. No big deal for now though :)